cli/command/container/restart.go

```diff
@@ -16,6 +16,8 @@ type restartOptions struct {
 	nSeconds        int
 	nSecondsChanged bool
 
+	nAll bool
```

Comment:
I couldn't tell the naming convention here so I went with it

Reply:
I think `all` is good enough. Don't know what the `n` prefix stands for, though.